src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -111,92 +107,92 @@ const CreateConceptForm = (props) => {
               )}
             </div>
           </div>
-          )}
-        </div>
-        <div className="datatypefield">
-          <label htmlFor="datatype">Datatype</label>
-          <select
-            id="datatype"
-            name="datatype"
-            required
-            value={props.state.datatype}
-            className="form-control "
-            onChange={props.handleChange}
-          >
-            <option>Numeric</option>
-            <option>Text</option>
-            <option>None</option>
-            <option>Document</option>
-            <option>Date</option>
-            <option>Time</option>
-            <option>Datetime</option>
-            <option>Boolean</option>
-            <option>Rule</option>
-            <option>Structured-Numeric</option>
-            <option>Complex</option>
-            <option>Coded</option>
-          </select>
-        </div>
+        )}
+      </div>
+      <div className="datatypefield">
+        <label htmlFor="datatype">Datatype</label>
+        <select
+          id="datatype"
+          name="datatype"
+          required
+          value={props.state.datatype}
+          className="form-control "
+          onChange={props.handleChange}
+        >
+          <option>Numeric</option>
+          <option>Text</option>
+          <option>None</option>
+          <option>Document</option>
+          <option>Date</option>
+          <option>Time</option>
+          <option>Datetime</option>
+          <option>Boolean</option>
+          <option>Rule</option>
+          <option>Structured-Numeric</option>
+          <option>Complex</option>
+          <option>Coded</option>
+        </select>
+      </div>
 
-        {concept.toString().trim() === 'question' ? (
+      {props.concept.toString().trim() === 'question'
+        ? (
           <div className="form-group answer">
             <div className="row col-12 custom-concept-list">
               <h6 className="text-left section-header">Answers</h6>
-              <AnswersTable
-                handleAsyncSelectChange={handleAsyncSelectChange}
-                queryAnswers={queryAnswers}
-                selectedAnswers={selectedAnswers}
-                handleAnswerChange={handleAnswerChange}
-              />
+              <AnswersTable {...props} />
+              <a href="#!" className="text-left add-more-answers" id="add-more-answers" onClick={props.addAnswer}>
+            Add a new answer mapping
```

Comment:
Where is the above line coming from?

Reply:
It was a previous version of the file, added mistakenly when I was fixing merge conflict.
I have updated it.